src/framework/mpas_stream_manager.F

```diff
@@ -4184,8 +4199,8 @@ subroutine build_stream(stream, direction, allFields, allPackages, timeLevelIn,
 
         integer :: local_ierr
 
-        integer, parameter :: idLength = 10
-        character (len=idLength) :: file_id
+        integer (c_int), parameter :: idLength = 10
+        character (len=idLength, kind=c_char) :: file_id
```

Comment:
For consistency with the declarations of C character strings elsewhere, I'd propose we change the declaration to:
```
character(kind=c_char), dimension(idLength) :: file_id
```